From 88790c5488fcfc1a9297d486ddeb63391401a640 Mon Sep 17 00:00:00 2001
From: Johannes Dewender <github@JonnyJD.net>
Date: Sun, 15 Sep 2013 14:22:46 +0200
Subject: [PATCH] fix MappedLun typo

Introduced in c37a00b93bff4d89b28992e7c4e6d1fa98351b74,
which is in fb38 and fb39.

The reported error was
NameError: global name 'MappedLun' is not defined.

Signed-off-by: Johannes Dewender <github@JonnyJD.net>
---
 rtslib/target.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rtslib/target.py b/rtslib/target.py
index 2ccdd43..da243c9 100644
--- a/rtslib/target.py
+++ b/rtslib/target.py
@@ -952,7 +952,7 @@ class NodeACL(CFSNode):
         set_attributes(acl_obj, acl.get('attributes', {}))
 
         for mlun in acl.get('mapped_luns', []):
-            MappedLun.setup(tpg_obj, acl_obj, mlun, err_func)
+            MappedLUN.setup(tpg_obj, acl_obj, mlun, err_func)
 
         dict_remove(acl, ('attributes', 'mapped_luns', 'node_wwn'))
         for name, value in acl.iteritems():
-- 
1.8.1.3

